### DISCUSSION TITLE
Dirty states with @entangle are not working

### DISCUSSION THREAD

**Discussion by Tylerian** [livewire/livewire#4428] [3. Feature ideas]
# Problem
Dirty states (`wire:dirty`) within an `@entangle`d property are not working properly. I've dived into the Livewire source code and it seems they're only triggered by the `input` event on an input element.

# Workaround
For now I'm using a dummy input and firing an `Event('input')` on it whenever I want to toggle the dirty state on a form but this is an ugly hack.

**Comment by joshhanley** [comment]
@Tylerian `wire:dirty` doesn't work with entangled properties, so I have changed this over to a feature request 🙂

**Comment by juliangarcess** [comment]
Hello!!, any fix?
